frontend/coprs_frontend/coprs/views/groups_ns/groups_general.py

```diff
@@ -41,12 +42,16 @@ def activate_group(fas_group):
         db.session.add(group)
         db.session.commit()
 
-        flask.flash(
-            "FAS group {} is activated in the Copr under the alias {} "
-            .format(fas_group, alias)
-        )
+        # can't have more than one alias for a group
+        if group.name != alias:
+            message = "Group {} already activated as alias {} ".format(
+                fas_group, group.name)
```

Comment:
If we go with error 400 or redirect, this `if/else` condition is redundant.  While on this, would you mind making the flash message "green" with `flash("message", "success")` :-) so it is more visible?  The grey color doesn't look like something actually happened...

Reply:
Done